.github/workflows/test-generate-contract-spec.yml

```diff
@@ -0,0 +1,123 @@
+---
+name: Test generate contract spec
+
+on:
+  pull_request:
+    branches:
+      - main
+      - A0-4070-add-generate-contract-spec
```

Comment:
This should be reverted before merge I guess

Reply:
Of course, and it's already removed.  I'll resolve this issue.